packages/amplify-cli-core/src/index.ts

```diff
@@ -253,7 +253,7 @@ interface AmplifyToolkit {
     filteredResources?: { category: string; resourceName: string }[],
   ) => $TSAny;
   storeCurrentCloudBackend: () => $TSAny;
-  readJsonFile: () => $TSAny;
+  readJsonFile: (fileName: string) => $TSAny;
```

Comment:
We could add a deprecated tag to this to encourage using stateManager instead

Reply:
Since it is mostly used from JS which is not even passed through the TS compile, nothing would warn :-(